src/lib/components/modal/bottomOrRegularModal/index.tsx

```diff
@@ -1,18 +1,42 @@
-import React from 'react';
-import { Props, RegularModal } from '..';
+import React, { useEffect, useRef, useState } from 'react';
 
+import { Props, RegularModal } from '..';
 import BottomModal from '../bottomModal';
 
 import styles from './style.module.scss';
 
-export default (props: Props) => {
+export default ({ isOpen, ...props }: Props) => {
+  const mobileRef = useRef<HTMLDivElement>(null);
+  const [visibleSize, setVisibleSize] = useState<'desktop' | 'mobile'>(
+    'desktop'
+  );
+
+  const handleResize = () => {
+    setVisibleSize(
+      mobileRef.current !== null &&
+        window.getComputedStyle(mobileRef.current).display !== 'none'
+        ? 'mobile'
+        : 'desktop'
+    );
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', handleResize);
+    handleResize();
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
   return (
     <>
-      <div className={styles.mobile}>
-        <BottomModal {...props} />
+      <div className={styles.mobile} ref={mobileRef}>
+        {visibleSize === 'mobile' && <BottomModal {...props} isOpen={isOpen} />}
       </div>
       <div className={styles.desktop}>
```

Comment:
Wouldn't this be an empty `div` if `visibleSize` is mobile?

Reply:
Perhaps `{visibleSize === 'desktop' && <div className={styles.desktop}><RegularModal /></div>}` is better?

Reply:
Yes good point